### ISSUE TITLE
fullconnect.py 中的Activation_Net 和 Batch_Net是一样的

### ISSUE THREAD

**Issue by bettermorn** [striver314/MSCNN#1]
代码注释说，在上面的Activation_Net的基础上，增加了一个加快收敛速度的方法——批标准化  ，但是代码里并没有体现出来。 Batch_Net的代码和Activation_Net是一样的，烦请解答。谢谢！

**Comment by striver314** [comment]
谢谢您的指正！代码中确实没有体现"批标准化"这一方法，现已更正（Batch_Net）如下：

`self.layer1 = nn.Sequential(nn.Linear(in_dim, n_hidden_1), nn.BatchNorm1d(n_hidden_1), nn.ReLU(True))`
`self.layer2 = nn.Sequential(nn.Linear(n_hidden_1, n_hidden_2), nn.BatchNorm1d(n_hidden_2), nn.ReLU(True))`
`self.layer3 = nn.Sequential(nn.Linear(n_hidden_2, out_dim))`

**Comment by bettermorn** [comment]
谢谢回复！
